OloEngine/src/OloEngine/Asset/AssetSerializer.cpp

```
@@ -59,6 +61,122 @@ namespace OloEngine
         return result;
     }
 
+    bool TextureSerializer::TryLoadRawData(const AssetMetadata& metadata, RawAssetData& outRawData) const
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)